.github/workflows/pr_push.yml

```diff
@@ -16,10 +16,17 @@ jobs:
                     - os: ubuntu-latest
                       nproc: $(nproc)
                       pool_scalable: 'ON'
+                      disjoint: 'ON'
+                      jemalloc: 'ON'
+                    - os: ubuntu-latest
```

Comment:
ugh, I think `include` does not work in this case as we would like it to 😉  We ended up only with 2 FastBuild jobs. I think, you have to add some param to the matrix (beside `os`) to overwrite its value in the `include`.

FYI, docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrixinclude

Reply:
That's odd. Refactored this a bit, let's see if it's any better.

Reply:
Now, there seem to be 2 ubuntu FastBuilds because of the `disjoint: ['ON', 'OFF']` matrix option. Couldn't we just run the job with disjoint pool set to `ON` and disregard the case where it's not built?

Reply:
I wanted to test exactly this case, where both disjoint pool and tests are turned off. That's the only case in our CI where you can verify a clean C build without CMake support for C++.